src/extension/hooks/useLocalStorage.jsx

```diff
@@ -0,0 +1,26 @@
+// https://usehooks.com/useLocalStorage/
```

Comment:
see comment from above, if we can make these utilities add our `ynab-toolkit-` prefix that would be ideal